pods-beaver-themer.php

```diff
@@ -80,6 +80,21 @@ function pods_beaver_admin_nag() {
 add_action( 'plugins_loaded', 'pods_beaver_admin_nag' );
 
 /**
+ * Enqueue assets
+ *
+ * @return void
+ *
+ * @since 1.1
+ */
+function pods_beaver_enqueue_assets() {
+    if ( FLBuilderModel::is_builder_active() ) {
```

Comment:
should we first check for: wp_doing_ajax() && 
as the settings panel is only loaded from ajax I guess we can't limit it to only posts modules can we?